sklearn/datasets/kddcup99.py

```diff
@@ -273,20 +276,26 @@ def _fetch_brute_kddcup99(data_home=None,
     else:
         # Backward compat for Python 2 users
         dir_suffix = ""
+
     if percent10:
         kddcup_dir = join(data_home, "kddcup99_10" + dir_suffix)
+        archive_path = join(kddcup_dir, ARCHIVE_10_PERCENT.filename)
+        expected_checksum = ARCHIVE_10_PERCENT.checksum
+        URL_ = ARCHIVE_10_PERCENT.url
     else:
         kddcup_dir = join(data_home, "kddcup99" + dir_suffix)
+        archive_path = join(kddcup_dir, ARCHIVE.filename)
+        expected_checksum = ARCHIVE.checksum
+        URL_ = ARCHIVE.url
+
     samples_path = join(kddcup_dir, "samples")
     targets_path = join(kddcup_dir, "targets")
     available = exists(samples_path)
 
     if download_if_missing and not available:
         _mkdirp(kddcup_dir)
-        URL_ = URL10 if percent10 else URL
-        logger.warning("Downloading %s" % URL_)
-        f = BytesIO(urlopen(URL_).read())
-
+        logger.info("Downloading %s" % URL_)
```

Comment:
This actually also happen [here](https://github.com/scikit-learn/scikit-learn/pull/9240/commits/7186af8e3291b810aab2659cf4459eb230864def#diff-ab16f11bd437de8db952d758adc6c615R82) , [here](https://github.com/scikit-learn/scikit-learn/pull/9240/commits/7186af8e3291b810aab2659cf4459eb230864def#diff-8d81a9148438e8d9515368c1a1574f74R87), [here](https://github.com/scikit-learn/scikit-learn/pull/9240/commits/7186af8e3291b810aab2659cf4459eb230864def#diff-bf281b3176e3a8090f0ec9f2ebca82e2R329), and [here](https://github.com/scikit-learn/scikit-learn/pull/9240/commits/7186af8e3291b810aab2659cf4459eb230864def#diff-ab16f11bd437de8db952d758adc6c615R82) 
I think they are all under the same case.